src/composition.rs

```diff
@@ -1142,41 +1262,57 @@ from(bucket: "my-bucket") |> yield(name: "my-result")
     }
 
     #[test]
-    fn composition_add_tag_tag_already_exists() {
```

Comment:
Just curious, why removing the test on a tag that is already exist?

Reply:
We could decide to leave it in. And error if it already exists.

But since we have a general rule (new rule) that the add_tag() and add_tag_value() always are append-only a new filter...I was not sure if we should still error. What do you think @rockstar ?

Reply:
Got it! Totally

Reply:
Now that I think about, if we allow 2 filters of the same tag...then we have to decide on the behavior of `remove_tag(tagName)`. Remove all filters for this tag? or do we only allow 1 in the first place?